src/select.coffee

```diff
@@ -166,8 +182,7 @@ Ember.AddeparMixins.ResizeHandlerMixin,
   # TODO(Peter): consider calling this optionViewClass?
   itemView: Ember.computed ->
     itemViewClass = @get 'itemViewClass'
-    if typeof itemViewClass is 'string'
-      return Ember.get itemViewClass
+    return Ember.get itemViewClass if typeof itemViewClass is 'string'
```

Comment:
Not a huge fan of this. This assumes that the string represents the global path to the class. But you didn't write it, so we can revisit that later.